rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -2004,6 +2012,54 @@ void should_SetTransferFlagToTrue_When_TransferringWithoutRequestBody() {
       assertThat(response.getBody().getWorkbasketSummary().getWorkbasketId())
           .isEqualTo("WBI:100000000000000000000000000000000006");
       assertThat(response.getBody().isTransferred()).isTrue();
+      assertThat(response.getBody().getOwner()).isNull();
+    }
+
+    @TestFactory
+    Stream<DynamicTest>
+        should_SetTransferFlagAndOwnerDependentOnRequestBody_When_TransferringTasks() {
```

Comment:
You only check a failed Task in this test. Why did you choose this test name, if both, owner and transferFlag, remain unchecken?

Reply:
Ok, I will change the name to "should_ReturnFailedTasks_When_TransferringTasks"